src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/DepartureEntity.kt

```diff
@@ -0,0 +1,43 @@
+package uk.gov.justice.digital.hmpps.approvedpremisesapi.jpa.entity
+
+import java.time.OffsetDateTime
+import java.util.UUID
+import javax.persistence.Entity
+import javax.persistence.EnumType
+import javax.persistence.Enumerated
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.OneToOne
+import javax.persistence.Table
+
+@Entity
+@Table(name = "departure")
+data class DepartureEntity(
+  @Id
+  var id: UUID,
+  var dateTime: OffsetDateTime,
+  @Enumerated(EnumType.STRING)
+  var reason: DepartureReason,
+  @Enumerated(EnumType.STRING)
+  var moveOnCategory: MoveOnCategory,
+  @ManyToOne
+  @JoinColumn(name = "destination_provider_id")
+  var destinationProvider: DestinationProviderEntity,
+  var notes: String?,
+  @OneToOne
+  @JoinColumn(name = "booking_id")
+  var booking: BookingEntity
+)
+
+enum class DepartureReason {
+  Absconded,
+  PlannedMove,
+  Recalled
+}
+
+enum class MoveOnCategory {
+  PrivateHouse,
+  Custody,
+  Deported
+}
```

Comment:
Could we add all the Reasons / Categories as per https://trello.com/c/0o9HkzJi/434-recording-departures#comment-62d98669beb5dc0130173739 pls?

Reply:
I'm also open to the idea of them being database entries - but I think enums is fine for now

Reply:
Given the use of commas, slashes, brackets etc. in a lot of the names I think we might be better off going for database entries - will make it a lot simpler if we need to reword some of them post-launch too since they can be keyed off a surrogate key rather than their literal names.

Reply:
Have pushed this change now